packages/ipfs/src/cli/commands/refs-local.js

```diff
@@ -1,27 +1,40 @@
 'use strict'
 
 const parseDuration = require('parse-duration')
+const multibase = require('multibase')
+const { Buffer } = require('buffer')
 
 module.exports = {
   command: 'refs-local',
 
   describe: 'List all local references.',
 
+  epilog: 'CIDs are reconstructed therefore they might differ from those under which the blocks were originally stored.',
+
   builder: {
     timeout: {
       type: 'string',
       coerce: parseDuration
+    },
+    multihash: {
+      type: 'boolean',
+      default: false,
+      desc: 'Shows base32 encoded multihashes instead of reconstructed CIDs'
     }
   },
 
-  async handler ({ ctx: { ipfs, print }, timeout }) {
-    for await (const ref of ipfs.refs.local({
+  async handler ({ ctx: { ipfs, print }, timeout, cidBase, multihash }) {
+    for await (const { ref, err } of ipfs.refs.local({
       timeout
     })) {
-      if (ref.err) {
-        print(ref.err, true, true)
+      if (err) {
+        print(err, true, true)
       } else {
-        print(ref.ref)
+        if (multihash) {
+          print(multibase.encode('base32', Buffer.from(ref)).toString().substring(1).toUpperCase())
```

Comment:
```suggestion
          print(multibase.encoding('base32upper').encode(Buffer.from(ref)))
```
with multibase v1 we can just do this or

```js
multibase.names['base32upper'].encode(Buffer.from(ref))
```

Reply:
Neato, I didn't realise.